types/underscore/underscore-tests.ts

```diff
@@ -474,6 +474,45 @@ _(explicitNumberDictionary).each((value, key, collection) => {
     _.chain(anyValue).forEach(anyCollectionIterator); // // $ExpectType _Chain<any, any>
 }
 
+// get
+
+{
+    // tslint:disable-next-line:no-inferrable-types
+    const num: number = 2;
+
+    // no default value
+    // $ExpectType number
+    _.get({ a: 1 }, 'a');
+
+    // trying to get the property on a simple path
+    // $ExpectType number
+    _.get({ a: 1 }, ['a'], num);
+
+    // trying to get the property on a simple path
+    // $ExpectType number
+    _.get({ a: 1 }, ['a'], num);
+
+    // trying to get the property one level deep
+    // $ExpectType number | { b: number; }
+    _.get({ a: { b: num } }, ['a'], num);
+
+    // trying to get the property one level deep
+    // $ExpectType number | { b: number; }
+    _.get({ a: { b: num } }, ['a', 'b'], num);
+
+    // default value if not found of the 'same type'
+    // $ExpectType number
+    _.get({ a: 1 }, ['b'], num);
+
+    // chained without default value
+    // $ExpectType unknown
+    _({ a: 1 }).get(['b']);
+
+    // chained with default value
```

Comment:
By the way, this is not chaining but OOP style. You have not tested the `_Chain` interface yet.